cluster/setup.go

```diff
@@ -67,8 +71,37 @@ func SetupM3Namespace() {
 	}
 }
 
-// Setups a postgres used by the provisioning service
-func SetupPostgres() {
+// SetupM3Secrets creates a kubernetes secrets
+func SetupM3Secrets() {
+	config := getConfig()
+	// creates the clientset
+	clientset, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		fmt.Println(err)
+		panic(err.Error())
+	}
+	// Create secret for JWT key for rest api
+	secret := v1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "jwtkey",
+		},
+		Data: map[string][]byte{
+			"M3_JWT_KEY": []byte(common.GetRandString(64, "default")),
+		},
+	}
+	res, err := clientset.CoreV1().Secrets("default").Create(&secret)
+	if err != nil {
+		fmt.Println(err)
```

Comment:
Error handling?

Reply:
So far we are using panic with all setups.